### ISSUE TITLE
GUI doesn't work in vanilla VirtualBox + Ubuntu 20.04

### ISSUE THREAD

**Issue by UnknownShadow200** [ClassiCube/MCGalaxy#658]
![image](https://user-images.githubusercontent.com/6509348/140643336-044dc9ec-f1af-416a-8464-6bdac6d46190.png)
See mono issue #13049 (System.ArgumentException: A null reference or invalid value was found [GDI+ status: InvalidParameter])

Some useful details
https://sourceforge.net/p/keepass/discussion/329221/thread/4ab5e244/#ef39
https://stackoverflow.com/questions/28674759/running-simple-winforms-application-on-mono-for-arm

Basically problem is XQueryBestCursor returns 0/0 under some circumstances which breaks cursor initialisation code. Not an easy issue to workaround
